nims-notes-things/src/components/Form.js

```diff
@@ -47,7 +61,21 @@ const Form = (props) => {
         id: uniqueId,
       };
       dispatch(setNote(formDataCopy));
-      alert("Note has been created :) ");
+      alert("Note has been created! :) ");
```

Comment:
Try to make a nice toast message instead of just alert. Check out [React Toastify](https://www.npmjs.com/package/react-toastify) with their cool toast builder [here](https://fkhadra.github.io/react-toastify/introduction/)

Reply:
Also once a note has been created or update you might want to redirect to the notes page?